src/Interface/Modules/Render/ViewScene.cc

```diff
@@ -186,19 +186,20 @@ namespace Gui {
 }}
 
 //--------------------------------------------------------------------------------------------------
-long PreviousWidgetSelectionInfo::timeSince(const std::chrono::system_clock::time_point& time) const
+unsigned long PreviousWidgetSelectionInfo::timeSince(const std::chrono::system_clock::time_point& time) const
 {
   return timeSinceEpoch(std::chrono::system_clock::now()) - timeSinceEpoch(time);
 }
 
 //--------------------------------------------------------------------------------------------------
-long PreviousWidgetSelectionInfo::timeSince(int time) const
+unsigned long PreviousWidgetSelectionInfo::timeSince(unsigned int time) const
 {
-  return long(int(timeSinceEpoch(std::chrono::system_clock::now())) -time);
+  unsigned int now = timeSinceEpoch(std::chrono::system_clock::now());
+  return  now - time;
 }
 
 //--------------------------------------------------------------------------------------------------
-long PreviousWidgetSelectionInfo::timeSinceEpoch(const std::chrono::system_clock::time_point& time) const
+unsigned long PreviousWidgetSelectionInfo::timeSinceEpoch(const std::chrono::system_clock::time_point& time) const
 {
   return std::chrono::duration_cast<std::chrono::milliseconds>(time.time_since_epoch()).count();
```

Comment:
chrono::milliseconds is "at least 45-bit integer" so you ought to change this to uint64_t rather than just unsigned long in order to avoid rollover bugs. Since it works on both of our machines for now, this can wait; maybe the windows problem was the use of the non-transient state.

Reply:
@RubioJr9